lib/p2p/node/ledger_manager.ex

```diff
@@ -54,7 +54,7 @@ defmodule Elixium.Node.LedgerManager do
       :ok ->
         # Save the block to our chain since its valid
         Ledger.append_block(block)
-        Oracle.inquire(:"Elixir.Elixium.Store.UtxoOracle", {:update_with_transactions, [block.transactions]})
+        Oracle.inquire(:"Elixir.Elixium.Store.UtxoOracle", {:update_with_transactions, [block.transactions]}, :infinity)
```

Comment:
Oracle.inquire/3 doesnt exist, since you've made :infinity a default

Reply:
Fixed